docs/toc.md

```diff
@@ -17,13 +17,13 @@
 ### [.NET Microservices: Architecture for Containerized .NET Applications](standard/microservices-architecture/)
 ## [Choosing between .NET Core and .NET Framework for server apps](standard/choosing-core-framework-server.md)
 ## [What is "managed code"?](standard/managed-code.md)
-### [Automatic Memory Management](standard/automatic-memory-management.md) 
+### [Automatic Memory Management](standard/automatic-memory-management.md)
 ## [Common Language Runtime (CLR)](standard/clr.md)
 ## [Language Independence](standard/language-independence.md)
 ### [Language Independence and Language-Independent Components](standard/language-independence-and-language-independent-components.md)
 ## [Framework Libraries](standard/framework-libraries.md)
-### [Class Library Overview](standard/class-library-overview.md)  
-### [Base Types](standard/base-types/)  
+### [Class Library Overview](standard/class-library-overview.md)
```

Comment:
Unrelated, but VS Code automatically strips trailing whitespace. I can remove if you want.